From khali@linux-fr.org Sun Aug 13 14:39:14 2006
Date: Sun, 13 Aug 2006 23:39:11 +0200
From: Arthur Othieno <apgo@patchbomb.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 07/12] i2c: Fix copy-n-paste in subsystem Kconfig
Message-Id: <20060813233911.da331c41.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-fix-copy-n-paste-in-subsystem-Kconfig.patch

From: Arthur Othieno <apgo@patchbomb.org>

i2c: Fix copy-n-paste in subsystem Kconfig

We have:
  drivers/i2c/Kconfig:2:# Character device configuration
Which is obviously not true..

Signed-off-by: Arthur Othieno <apgo@patchbomb.org>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/i2c/Kconfig
+++ gregkh-2.6/drivers/i2c/Kconfig
@@ -1,5 +1,5 @@
 #
-# Character device configuration
+# I2C subsystem configuration
 #
 
 menu "I2C support"
